pages/dao/[symbol]/index.tsx

```diff
@@ -34,9 +34,9 @@ const REALM = () => {
   const [filteredProposals, setFilteredProposals] = useState(displayedProposals)
   const wallet = useWalletStore((s) => s.current)
 
-  const allProposals = Object.entries(proposals)
-    .filter(([, v]) => v.info.votingAt)
-    .sort((a, b) => compareProposals(b[1].info, a[1].info))
+  const allProposals = Object.entries(proposals).sort((a, b) =>
+    compareProposals(b[1].info, a[1].info)
+  )
```

Comment:
@saml33 not sure why this was filtering on votingAt? should that be handled in the filters if desired?